marklogic-data-hub-spark-connector/spark-test-project/src/main/java/test/StreamTest.java

```diff
@@ -0,0 +1,96 @@
+package test;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.streaming.StreamingQueryException;
+import org.apache.spark.sql.types.StructType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+public class StreamTest {
+
+    private static Logger logger = LoggerFactory.getLogger(StreamTest.class);
+
+    private static String host = "localhost";
+    private static String username = "test-data-hub-operator";
+    private static String password = "password";
+
+    public static void main(String[] args) throws Exception {
+        setConnectionProperties(args);
+
+        logger.info("Creating SparkSession");
+
+        SparkSession sparkSession = SparkSession.builder()
+            .master("local")
+            .getOrCreate();
+
+
+        try {
+            Dataset<Row> rows = loadRowsFromTestFile(sparkSession);
+            rows.join(rows);
```

Comment:
Can you remove this line from both test programs? I think I included it via copy/paste, but it doesn't seem necessary, and we want as few lines of code here as possible.